playbooks/configure_network.yml

```diff
@@ -1,5 +1,13 @@
 ---
-
+- name: Check remote connections
+  hosts: "{{ edpm_override_hosts | default('all', true) }}"
+  strategy: linear
+  gather_facts: "{{ gather_facts | default(false) }}"
+  tasks:
```

Comment:
If decide to go this route, then we also should add an ansible var to run the task only for bgp deployments.